test/com/google/enterprise/adaptor/database/UniqueKeyTest.java

```diff
@@ -317,15 +319,15 @@ public void testPreparingRetrieval_wrongColumnCount() throws SQLException {
   @Test
   public void testPreparingRetrieval() throws SQLException {
     executeUpdate("create table data("
-        + "numnum int, strstr varchar, timestamp timestamp, date date, "
-        + "time time, long bigint)");
+        + "numnum int, strstr varchar(20), tymestamp timestamp(3), date date, "
```

Comment:
Timestamp isn't an issue with the SQL, but it interfered with my create table rewriting of timestamp in JdbcFixture. However the "long" on the next line did run afoul of MySQL, which considers it a modifier of other data types.